oximeter/db/src/client/dbwrite.rs

```diff
@@ -59,11 +59,20 @@ impl DbWrite for Client {
     }
 
     /// Initialize the replicated telemetry database, creating tables as needed.
+    ///
+    /// We run both db-init files since we want all tables in production.
+    /// These files are intentionally disjoint so that we don't have to
+    /// duplicate any setup.
     async fn init_replicated_db(&self) -> Result<(), Error> {
         debug!(self.log, "initializing ClickHouse database");
         self.run_many_sql_statements(include_str!(concat!(
             env!("CARGO_MANIFEST_DIR"),
-            "/schema/replicated/db-init.sql"
+            "/schema/replicated/db-init-1.sql"
```

Comment:
Just checking, this split of the SQL files is only used for the replicated tables, right?

Reply:
Yep.